javascript/modules/client/src/client.ts

```diff
@@ -119,12 +172,34 @@ export class Client extends ClientCore implements IClient {
 
   /** Contains the gas estimation of the Ethereum transactions */
   estimation = {
+    /**
+     * Estimates the gas fee of creating a DAO
+     *
+     * @param {ICreateParams} params
+     * @return {*}  {Promise<GasFeeEstimation>}
+     * @memberof Client
+     */
     create: (params: ICreateParams) => this._estimateCreation(params),
+
+    /**
+     * Estimates the gas fee of depositing ether or an ERC20 token into the dao
+     *
+     * @param {IDepositParams} params
+     * @return {*}  {Promise<GasFeeEstimation>}
+     * @memberof Client
+     */
     deposit: (params: IDepositParams) => this._estimateDeposit(params),
   };
 
   //// PRIVATE METHOD IMPLEMENTATIONS
-
+  /**
+   *
+   *
+   * @private
+   * @param {ICreateParams} params
+   * @return {*}  {AsyncGenerator<DaoCreationStepValue>}
+   * @memberof Client
+   */
   private async *_createDao(
```

Comment:
We might not need that on private functions, but it's nice nonetheless

Reply:
I'm going to delete it for consistency, if we want to add jsdoc on private functions it should be a different task, and we should add it everywhere

Reply:
I don't see it deleted